docs/running-calyx/fud2/scripts.md

```diff
@@ -28,6 +28,66 @@ export const verilog_state = state("verilog", ["sv", "v"]);
 
 These two lines define a `calyx` state and a `verilog` state. The `export` prefix means that these variables will be accessible to other scripts that `import "calyx"`.
 
+Now we will define an operation taking the `calyx` state to the `verilog` state. These operations define functions where the signiture asserts the result of running the body.
```

Comment:
Not sure I understand "asserts the result of running the body." Maybe we just want to say something like: "Operations are functions whose arguments are input files and return values are output values. Their bodies consist of commands that will transform those inputs to those outputs."